numba/typed/typedobjectutils.py

```diff
@@ -120,15 +120,16 @@ def _container_get_meminfo(context, builder, container_ty, c):
     return conatainer_struct.meminfo
 
 
-def _get_incref_decref(context, module, datamodel, container_type):
+def _get_incref_decref(context, module, datamodel, container_element_type):
     assert datamodel.contains_nrt_meminfo()
 
     fe_type = datamodel.fe_type
     data_ptr_ty = datamodel.get_data_type().as_pointer()
     refct_fnty = ir.FunctionType(ir.VoidType(), [data_ptr_ty])
     incref_fn = module.get_or_insert_function(
         refct_fnty,
-        name='.numba_{}_incref${}'.format(container_type, fe_type),
+        name='{}.{}_incref'.format(
```

Comment:
```suggestion
        name='.numba_{}.{}_incref'.format(
```
it's helpful to keep the `numba` name in there for debugging.